.vib/couchdb/goss/couchdb.yaml

```
@@ -0,0 +1,16 @@
+file:
+  # Checks default config for erlang is set
+  {{ .Vars.root_dir }}/couchdb/etc/vm.args:
```

```suggestion
  /opt/bitnami/couchdb/etc/vm.args:
```

IIRC, we agreed on using the full path for file-related tests in the past